app/scripts/utils/table-config/collections.js

```diff
@@ -42,6 +42,19 @@ export const tableSortProps = [
   'timestamp'
 ];
 
+const confirmRecover = (d) => `Recover all granules from ${d} ${strings.collection}(s)?`;
```

Comment:
```suggestion
const confirmRecover = (d) => `Recover all collections from ${d} ${strings.collection}(s)?`;
```

Reply:
Out of curiosity, why is the confirmation message here not just `Recover ${d} ${strings.collections}(s)?`

Reply:
Good question. I think that's a good idea. Ideally the users are going to define their own recovery workflow, so we don't really have insight into what it means to "recover a collection." Probably a good idea to let the user fill in that blank.

Reply:
https://github.com/nasa/cumulus-dashboard/pull/569/commits/f73ab300344d599e3f86325cdc989f627a6e5206